.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy MkDocs to gh-pages
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variable
+        id: plum
+        run: echo "::set-output name=PULL_NUMBER::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build MkDocs
+        uses: jnanadarshan/mkdocs-build-action@v0.24
```

Comment:
@KeisukeShima By the way why do you use this action instead of just using `pip`? :thinking:

Reply:
@kenji-miyake I replaced script with pip :+1: Now it looks fine.